src/task/script.js

```diff
@@ -418,3 +433,11 @@ assigneeEl.addEventListener(
   'input',
   debounce((event) => fetchMembers(event.target.value), 500),
 );
+
+function addDisplay(element) {
+  element.classList.remove('element-display-remove');
+}
+
+function removeDisplay(element) {
```

Comment:
Change the name to `removeClassName` to make it more generic.

Move these functions to the utils.js file and export from there

Reply:
updated the function name, as someone is already working on code restructure so once the code gets restructured will add this function to the utils file